go/libkb/secret_store.go

```diff
@@ -129,6 +129,39 @@ func GetConfiguredAccounts(m MetaContext, s SecretStoreAll) ([]keybase1.Configur
 			IsCurrent: username.Eq(currentUsername),
 		}
 	}
+
+	// Get the full names
+
+	uids := make([]keybase1.UID, 0, len(allUsernames))
+	uidMapper := m.G().UIDMapper
+	for _, username := range allUsernames {
+		uid := uidMapper.MapHardcodedUsernameToUID(username)
+		if !uid.Exists() {
+			uid = UsernameToUIDPreserveCase(username.String())
+		}
+		uids = append(uids, uid)
+	}
+	usernamePackages, err := uidMapper.MapUIDsToUsernamePackages(m.Ctx(), m.G(),
+		uids, time.Hour*24, time.Second*10, false)
+	if err != nil {
+		if usernamePackages != nil {
+			// If data is returned, interpret the error as a warning
+			m.G().Log.CInfof(m.Ctx(),
+				"error while retreiving full names: %+v", err)
+		} else {
+			return nil, err
+		}
+	}
+	for _, uPackage := range usernamePackages {
+		account, ok := accounts[uPackage.NormalizedUsername]
+		if ok && (uPackage.FullName != nil) {
```

Comment:
`()` is redundant, I'd write it like this

```
for _, upkg := range usernamePackages {
	if account, ok := accounts[upkg.NormalizedUsername]; ok && upkg.FullName != nil {
		accounts[upkg.NormalizedUsername].Fullname = upkg.FullName.FullName
	}
}
```

Reply:
FWIW either of my suggestions wouldn't compile because `account` isn't used. My concern here was mostly the copying rather than setting a field directly in the struct. Not that it matters performance-wise, it just looks off.

So:
```go
if _, ok := accounts[uPackage.NormalizedUsername]; ok {
	accounts[uPackage.NormalizedUsername].Fullname = uPackage.FullName.FullName
}
```

Reply:
I'm pretty sure ```	accounts[uPackage.NormalizedUsername].Fullname = uPackage.FullName.FullName``` is not allowed by the compiler